grafana-plugin/.config/webpack/webpack.config.ts

```diff
@@ -199,7 +240,7 @@ const config = async (env): Promise<Configuration> => {
 
   if (isWSL()) {
     baseConfig.watchOptions = {
-      poll: 3000,
+      // poll: 3000,
```

Comment:
on WSL this isn't really needed. I use WSL and this breaks reloading actually, doesn't seem to be configurable :(